core/server/worker/src/test/java/alluxio/worker/block/allocator/MaxFreeAllocatorTest.java

```diff
@@ -103,7 +103,7 @@ public void allocateBlock() throws Exception {
     //  1               ├─── 3000
     //  2               └─── 3000
```

Comment:
Can you please also update the diagram in the comments to reflect the correct allocation results after this bug is fixed?

Reply:
Fixed, please help check.